lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1923,6 +1923,17 @@ def _resource_sku_to_capability(  # noqa: C901
             node_space.network_interface.max_nic_count = 1
             node_space.network_interface.nic_count = search_space.IntRange(min=1, max=1)
 
+        assert resource_sku.family, "'resource_sku.family' must not be 'None'"
```

Comment:
It should be in feature's `create_setting`. If it's not supported, return None. Refer to Gpu feature.